packages/devtools-source-map/src/worker.js

```diff
@@ -1,234 +1,19 @@
-    // request.
-    return existingRequest;
-  } else if (!generatedSource.sourceMapURL || !sourceMapsEnabled) {
-    return Promise.resolve(null);
-  }
-
-  // Fire off the request, set it in the cache, and return it.
-  const req = _resolveAndFetch(generatedSource).catch(e => console.error(e));
-  sourceMapRequests.set(generatedSource.id, req);
-  return req;
-}
-
-async function getOriginalURLs(generatedSource) {
-  const map = await _fetchSourceMap(generatedSource);
-  return map && map.sources;
-}
-
-async function getGeneratedLocation(location: Location, originalSource: Source)
-    : Promise<Location> {
-  if (!isOriginalId(location.sourceId)) {
-    return location;
-  }
-
-  const generatedSourceId = originalToGeneratedId(location.sourceId);
-  const map = await _getSourceMap(generatedSourceId);
-  if (!map) {
-    return location;
-  }
-
-  const { line, column } = map.generatedPositionFor({
-    source: originalSource.url,
-    line: location.line,
-    column: location.column == null ? 0 : location.column,
-    bias: SourceMapConsumer.LEAST_UPPER_BOUND
-  });
-
-  return {
-    sourceId: generatedSourceId,
-    line: line,
-    // Treat 0 as no column so that line breakpoints work correctly.
-    column: column === 0 ? undefined : column
-  };
-}
-
-async function getOriginalLocation(location: Location) {
-  if (!isGeneratedId(location.sourceId)) {
-    return location;
-  }
-
-  const map = await _getSourceMap(location.sourceId);
-  if (!map) {
-    return location;
-  }
-
-  const { source: url, line, column } = map.originalPositionFor({
-    line: location.line,
-    column: location.column == null ? Infinity : location.column
-  });
-
-  if (url == null) {
-    // No url means the location didn't map.
-    return location;
-  }
-
-  return {
-    sourceId: generatedToOriginalId(location.sourceId, url),
-    line,
-    column
-  };
-}
-
-async function getOriginalSourceText(originalSource: Source) {
-  assert(isOriginalId(originalSource.id),
-         "Source is not an original source");
-
-  const generatedSourceId = originalToGeneratedId(originalSource.id);
-  const map = await _getSourceMap(generatedSourceId);
-  if (!map) {
-    return null;
-  }
-
-  let text = map.sourceContentFor(originalSource.url);
-  if (!text) {
-    text = (await networkRequest(
-      originalSource.url, { loadFromCache: false }
-    )).content;
-  }
-
-  return {
-    text,
-    contentType: getContentType(originalSource.url || "")
-  };
-}
-
-function applySourceMap(generatedId, url, code, mappings) {
-  const generator = new SourceMapGenerator({ file: url });
-  mappings.forEach(mapping => generator.addMapping(mapping));
-  generator.setSourceContent(url, code);
-
-  const map = SourceMapConsumer(generator.toJSON());
-  sourceMapRequests.set(generatedId, Promise.resolve(map));
-}
-
-const publicInterface = {
   getOriginalURLs,
   getGeneratedLocation,
   getOriginalLocation,
   getOriginalSourceText,
-  enableSourceMaps,
```

Comment:
Hmm, I am a bit perplexed by `enableSourceMaps`... It doesn't look like debugger is calling it even today? So... how are source maps working...? 😕 Are you removing it because it's unused?